components/eppp_link/eppp_link.c

```diff
@@ -703,7 +706,7 @@ void eppp_deinit(esp_netif_t *netif)
 
 esp_netif_t *eppp_init(eppp_type_t role, eppp_config_t *config)
 {
-    esp_netif_t *netif = netif_init(role);
+    esp_netif_t *netif = netif_init(role, config);
```

Comment:
Shall we verify if `role` and `config` are not `NULL`?

Reply:
Have added null checks to the public APIs:
* `eppp_init()`
* `eppp_open()`